CHANGELOG.md

```diff
@@ -15,6 +15,10 @@
   made slash-free in both cases. This is a behavioral change, but it's unlikely
   to affect any real-world stylesheets.
 
+* [`:is()`][] now behaves identically to `:matches()`.
+
+  [`:is()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/:is
```

Comment:
It's a good idea to put this at the root level rather than indented to match the bullet point; although it *should* work either way, some Markdown processors choke on indented references.